packages/plugin-amplitude/src/androidMain/kotlin/ly.iterative.itly.amplitude/AmplitudeOptions.kt

```diff
@@ -3,5 +3,34 @@ package ly.iterative.itly.amplitude
 import android.content.Context
 
 actual open class AmplitudeOptions(
```

Comment:
@qingzhuozhen For Java it might be good to add ` @JvmOverloads` here so all the user won't need to set all options in the constructor https://medium.com/android-news/demystifying-the-jvmoverloads-in-kotlin-10dd098e6f72

Reply:
Here's an example in the repo - https://github.com/amplitude/itly-sdk-jvm/blob/master/packages/sdk-core/src/main/kotlin/ly.iterative.itly/Options.kt